index.js

```diff
@@ -208,6 +208,18 @@ Proxy.prototype = {
         this.doReqWithOptions(options, postData, cb);
     },
 
+    /*
+     *  matchRegex - A matching URL regular expression
+     *  replace - Replacement URL
+     */
+    rewrite: function(port, obj, cb) {
+        var data = '';
```

Comment:
does this need to be:

var data = '?'
to start the query string?

Reply:
I followed the example of `limit` method:
https://github.com/zzo/browsermob-node/blob/392e9cd81355d4d27d769c0230322ad38bfe0398/index.js#L154-L157

Works nicely for me, checked on a live project.